events/elections/2020/README.md

```diff
@@ -0,0 +1,169 @@
+# 2020 VOTERS GUIDE - KUBERNETES STEERING COMMITTEE ELECTION
+
+## Purpose
+
+The role of this election is to fill out the three (3) seats due for
+reelection this year on the [Kubernetes Steering Committee]. Each elected
+member will serve a two (2) year term.
+
+## Background
+
+This election will shape the future of Kubernetes as a community and project.
```

Comment:
This election will help shape the future governance policies for the Kubernetes community and project.